### PR TITLE
[Tizen] Tizen:ime tag parsing added

### PR DESCRIPTION
**Pull request by karolpawlowski**
According to specification https://source.tizen.org/sites/default/files/page/tizen-2.3-wrt-core-spec.pdf
the new tag (tizen:ime) was added and widget with this tag should have specified
tizen:category with name="http://tizen.org/category/ime"

BUG=XWALK-3135

### CODE REVIEW

**Comment by karolpawlowski** [comment]
@pozdnyakov I hope its done now ;)

File: application/common/manifest_handlers/tizen_ime_handler.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+namespace xwalk {
+namespace application {
+
+class TizenImeInfo : public ApplicationData::ManifestData {
+ public:
+  TizenImeInfo();
+  virtual ~TizenImeInfo();
+
+  const std::string& Uuid() const {
```
Comment (pozdnyakov) [inline application/common/manifest_handlers/tizen_ime_handler.h]: s/Uuid/uuid

File: application/common/manifest_handlers/tizen_ime_handler.h [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    return uuid_;
+  }
+  inline void SetUuid(const std::string& uuid) { uuid_ = uuid; }
```
Comment (pozdnyakov) [inline application/common/manifest_handlers/tizen_ime_handler.h]: s/SetUuid/set_uid (as far as I remember I had mentioned it before.)
also pls remove the "inline" keyword

File: application/common/manifest_handlers/tizen_ime_handler.h [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  const std::vector<std::string>& Languages() const {
```
Comment (pozdnyakov) [inline application/common/manifest_handlers/tizen_ime_handler.h]: s/Languages/languages

File: application/common/manifest_handlers/tizen_ime_handler.cc [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+TizenImeHandler::~TizenImeHandler() {
+}
+
+void TizenImeInfo::AddLanguage(const std::string& language) {
+  languages_.push_back(language);
+}
+
+bool TizenImeHandler::Parse(scoped_refptr<ApplicationData> application,
+    base::string16* error) {
+  scoped_ptr<TizenImeInfo> imeInfo(new TizenImeInfo);
```
Comment (pozdnyakov) [inline application/common/manifest_handlers/tizen_ime_handler.cc]: ime_info

**Comment by pozdnyakov** [comment]
lgtm with nits

**Comment by karolpawlowski** [comment]
@rakuco what is your opinion?

**Comment by rakuco** [comment]
I trust whatever Mikhail says :-)

**Pull request merged**